vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/BigDecimalField.java

```diff
@@ -309,6 +308,11 @@ public void setAutoselect(boolean autoselect) {
         super.setAutoselect(autoselect);
     }
 
+    @Override
+    public void setAutofocus(boolean autofocus) {
+        super.setAutofocus(autofocus);
+    }
+
```

Comment:
What's the reason for overriding this method? Is it for defining a custom javadoc? If so, then it's missing.

Reply:
@sissbruecker has re-added this, because I accidentally deleted this while merging this old PR to the latest master two weeks ago. If you look at the total diff, the diff / addition isn't there. 

See master: https://github.com/vaadin/flow-components/blob/6ec2a0aae54f3f042995605903faf04c81174a38/vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/BigDecimalField.java#L331-L335

Reply:
@DiegoCardoso Yeah, this is just something I restored from a previous commit: https://github.com/vaadin/flow-components/pull/2320/commits/d4218c9fd9fcc8413a6a9dca7ac99ef9d67f10f2#diff-4470d1fe00e1d9191825db12d8da376803057f97da2c5d15f546b7006ba0adb4. It should not result in a change to master.

Reply:
Got it. I was checking the commit done by @sissbruecker, so that the source of confusion. I still don't understand why we have this method overridden here, but that's another thing.